pages/user/index.vue

```diff
@@ -6,8 +6,29 @@
 
   const authStore = useAuthStore()
   const { user } = storeToRefs(authStore)
+
+  const localePath = useLocalePath()
+  const goToEdit = () => {
+    // asume normal user
+    navigateTo(localePath(`/user/edit/${user?.id}`))
```

Comment:
I couldnt get this to work as it renders correct data, but the url states `user/edit/undefined`. I would explicitly set the @click in the `<VBtn>` to 
```vue
<VBtn
  ...
  @click="navigateTo(localePath(`/user/edit/${user?.id}`))"
>
  ...
</VBtn>
```

Reply:
should be fixed